modRlp/src/main/java/org/aion/rlp/RLP.java

```diff
@@ -67,6 +67,11 @@
      */
     private static final int OFFSET_LONG_LIST = 0xf7;
 
+    private static final byte[] BYTES_SHORT_ITEM = new byte[] {(byte)0x80};
```

Comment:
I'd replace `(byte)0x80` with `(byte) OFFSET_SHORT_ITEM`; also class formatting may be off

Reply:
updated